src/cluster/cluster.cc

```diff
@@ -319,7 +319,8 @@ Status Cluster::ImportSlot(Redis::Connection *conn, int slot, int state) {
       conn->SetImporting();
       myself_->importing_slot_ = slot;
       // Set link error callback
-      conn->close_cb_ = std::bind(&SlotImport::StopForLinkError, svr_->slot_import_, conn->GetFD());
+      // conn->close_cb_ = std::bind(&SlotImport::StopForLinkError, svr_->slot_import_, conn->GetFD());
+      conn->close_cb_ = [ObjectPtr = svr_->slot_import_, captureFd = conn->GetFD()](int fd) {ObjectPtr->StopForLinkError(captureFd);};
```

Comment:
To be consistent, please use variables `object_ptr` (snake case) instead of `ObjectPtr` (camel case). `captureFd` as well.